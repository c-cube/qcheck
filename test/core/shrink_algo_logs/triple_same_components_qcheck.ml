open QCheck

let triple_same =
  Test.make ~name:"triples have same components"
    (triple int int int)
    (Log.shrinks Print.(triple int int int) @@ fun (i,j,k) -> i=j || j=k)
;;
try Test.check_exn triple_same with Test.Test_fail _ -> ()
