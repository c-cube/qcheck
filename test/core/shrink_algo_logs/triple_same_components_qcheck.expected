fails (3367339764960236310, -4202639126704614816, 2724675603984413065)
fails (1683669882480118155, -4202639126704614816, 2724675603984413065)
fails (841834941240059078, -4202639126704614816, 2724675603984413065)
fails (420917470620029539, -4202639126704614816, 2724675603984413065)
fails (210458735310014770, -4202639126704614816, 2724675603984413065)
fails (105229367655007385, -4202639126704614816, 2724675603984413065)
fails (52614683827503693, -4202639126704614816, 2724675603984413065)
fails (26307341913751847, -4202639126704614816, 2724675603984413065)
fails (13153670956875924, -4202639126704614816, 2724675603984413065)
fails (6576835478437962, -4202639126704614816, 2724675603984413065)
fails (3288417739218981, -4202639126704614816, 2724675603984413065)
fails (1644208869609491, -4202639126704614816, 2724675603984413065)
fails (822104434804746, -4202639126704614816, 2724675603984413065)
fails (411052217402373, -4202639126704614816, 2724675603984413065)
fails (205526108701187, -4202639126704614816, 2724675603984413065)
fails (102763054350594, -4202639126704614816, 2724675603984413065)
fails (51381527175297, -4202639126704614816, 2724675603984413065)
fails (25690763587649, -4202639126704614816, 2724675603984413065)
fails (12845381793825, -4202639126704614816, 2724675603984413065)
fails (6422690896913, -4202639126704614816, 2724675603984413065)
fails (3211345448457, -4202639126704614816, 2724675603984413065)
fails (1605672724229, -4202639126704614816, 2724675603984413065)
fails (802836362115, -4202639126704614816, 2724675603984413065)
fails (401418181058, -4202639126704614816, 2724675603984413065)
fails (200709090529, -4202639126704614816, 2724675603984413065)
fails (100354545265, -4202639126704614816, 2724675603984413065)
fails (50177272633, -4202639126704614816, 2724675603984413065)
fails (25088636317, -4202639126704614816, 2724675603984413065)
fails (12544318159, -4202639126704614816, 2724675603984413065)
fails (6272159080, -4202639126704614816, 2724675603984413065)
fails (3136079540, -4202639126704614816, 2724675603984413065)
fails (1568039770, -4202639126704614816, 2724675603984413065)
fails (784019885, -4202639126704614816, 2724675603984413065)
fails (392009943, -4202639126704614816, 2724675603984413065)
fails (196004972, -4202639126704614816, 2724675603984413065)
fails (98002486, -4202639126704614816, 2724675603984413065)
fails (49001243, -4202639126704614816, 2724675603984413065)
fails (24500622, -4202639126704614816, 2724675603984413065)
fails (12250311, -4202639126704614816, 2724675603984413065)
fails (6125156, -4202639126704614816, 2724675603984413065)
fails (3062578, -4202639126704614816, 2724675603984413065)
fails (1531289, -4202639126704614816, 2724675603984413065)
fails (765645, -4202639126704614816, 2724675603984413065)
fails (382823, -4202639126704614816, 2724675603984413065)
fails (191412, -4202639126704614816, 2724675603984413065)
fails (95706, -4202639126704614816, 2724675603984413065)
fails (47853, -4202639126704614816, 2724675603984413065)
fails (23927, -4202639126704614816, 2724675603984413065)
fails (11964, -4202639126704614816, 2724675603984413065)
fails (5982, -4202639126704614816, 2724675603984413065)
fails (2991, -4202639126704614816, 2724675603984413065)
fails (1496, -4202639126704614816, 2724675603984413065)
fails (748, -4202639126704614816, 2724675603984413065)
fails (374, -4202639126704614816, 2724675603984413065)
fails (187, -4202639126704614816, 2724675603984413065)
fails (94, -4202639126704614816, 2724675603984413065)
fails (47, -4202639126704614816, 2724675603984413065)
fails (24, -4202639126704614816, 2724675603984413065)
fails (12, -4202639126704614816, 2724675603984413065)
fails (6, -4202639126704614816, 2724675603984413065)
fails (3, -4202639126704614816, 2724675603984413065)
fails (2, -4202639126704614816, 2724675603984413065)
fails (1, -4202639126704614816, 2724675603984413065)
fails (0, -4202639126704614816, 2724675603984413065)
fails (0, -2101319563352307408, 2724675603984413065)
fails (0, -1050659781676153704, 2724675603984413065)
fails (0, -525329890838076852, 2724675603984413065)
fails (0, -262664945419038426, 2724675603984413065)
fails (0, -131332472709519213, 2724675603984413065)
fails (0, -65666236354759607, 2724675603984413065)
fails (0, -32833118177379804, 2724675603984413065)
fails (0, -16416559088689902, 2724675603984413065)
fails (0, -8208279544344951, 2724675603984413065)
fails (0, -4104139772172476, 2724675603984413065)
fails (0, -2052069886086238, 2724675603984413065)
fails (0, -1026034943043119, 2724675603984413065)
fails (0, -513017471521560, 2724675603984413065)
fails (0, -256508735760780, 2724675603984413065)
fails (0, -128254367880390, 2724675603984413065)
fails (0, -64127183940195, 2724675603984413065)
fails (0, -32063591970098, 2724675603984413065)
fails (0, -16031795985049, 2724675603984413065)
fails (0, -8015897992525, 2724675603984413065)
fails (0, -4007948996263, 2724675603984413065)
fails (0, -2003974498132, 2724675603984413065)
fails (0, -1001987249066, 2724675603984413065)
fails (0, -500993624533, 2724675603984413065)
fails (0, -250496812267, 2724675603984413065)
fails (0, -125248406134, 2724675603984413065)
fails (0, -62624203067, 2724675603984413065)
fails (0, -31312101534, 2724675603984413065)
fails (0, -15656050767, 2724675603984413065)
fails (0, -7828025384, 2724675603984413065)
fails (0, -3914012692, 2724675603984413065)
fails (0, -1957006346, 2724675603984413065)
fails (0, -978503173, 2724675603984413065)
fails (0, -489251587, 2724675603984413065)
fails (0, -244625794, 2724675603984413065)
fails (0, -122312897, 2724675603984413065)
fails (0, -61156449, 2724675603984413065)
fails (0, -30578225, 2724675603984413065)
fails (0, -15289113, 2724675603984413065)
fails (0, -7644557, 2724675603984413065)
fails (0, -3822279, 2724675603984413065)
fails (0, -1911140, 2724675603984413065)
fails (0, -955570, 2724675603984413065)
fails (0, -477785, 2724675603984413065)
fails (0, -238893, 2724675603984413065)
fails (0, -119447, 2724675603984413065)
fails (0, -59724, 2724675603984413065)
fails (0, -29862, 2724675603984413065)
fails (0, -14931, 2724675603984413065)
fails (0, -7466, 2724675603984413065)
fails (0, -3733, 2724675603984413065)
fails (0, -1867, 2724675603984413065)
fails (0, -934, 2724675603984413065)
fails (0, -467, 2724675603984413065)
fails (0, -234, 2724675603984413065)
fails (0, -117, 2724675603984413065)
fails (0, -59, 2724675603984413065)
fails (0, -30, 2724675603984413065)
fails (0, -15, 2724675603984413065)
fails (0, -8, 2724675603984413065)
fails (0, -4, 2724675603984413065)
fails (0, -2, 2724675603984413065)
fails (0, -1, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, -1, 1362337801992206533)
holds (0, 0, 1362337801992206533)
fails (0, -1, 681168900996103267)
holds (0, 0, 681168900996103267)
fails (0, -1, 340584450498051634)
holds (0, 0, 340584450498051634)
fails (0, -1, 170292225249025817)
holds (0, 0, 170292225249025817)
fails (0, -1, 85146112624512909)
holds (0, 0, 85146112624512909)
fails (0, -1, 42573056312256455)
holds (0, 0, 42573056312256455)
fails (0, -1, 21286528156128228)
holds (0, 0, 21286528156128228)
fails (0, -1, 10643264078064114)
holds (0, 0, 10643264078064114)
fails (0, -1, 5321632039032057)
holds (0, 0, 5321632039032057)
fails (0, -1, 2660816019516029)
holds (0, 0, 2660816019516029)
fails (0, -1, 1330408009758015)
holds (0, 0, 1330408009758015)
fails (0, -1, 665204004879008)
holds (0, 0, 665204004879008)
fails (0, -1, 332602002439504)
holds (0, 0, 332602002439504)
fails (0, -1, 166301001219752)
holds (0, 0, 166301001219752)
fails (0, -1, 83150500609876)
holds (0, 0, 83150500609876)
fails (0, -1, 41575250304938)
holds (0, 0, 41575250304938)
fails (0, -1, 20787625152469)
holds (0, 0, 20787625152469)
fails (0, -1, 10393812576235)
holds (0, 0, 10393812576235)
fails (0, -1, 5196906288118)
holds (0, 0, 5196906288118)
fails (0, -1, 2598453144059)
holds (0, 0, 2598453144059)
fails (0, -1, 1299226572030)
holds (0, 0, 1299226572030)
fails (0, -1, 649613286015)
holds (0, 0, 649613286015)
fails (0, -1, 324806643008)
holds (0, 0, 324806643008)
fails (0, -1, 162403321504)
holds (0, 0, 162403321504)
fails (0, -1, 81201660752)
holds (0, 0, 81201660752)
fails (0, -1, 40600830376)
holds (0, 0, 40600830376)
fails (0, -1, 20300415188)
holds (0, 0, 20300415188)
fails (0, -1, 10150207594)
holds (0, 0, 10150207594)
fails (0, -1, 5075103797)
holds (0, 0, 5075103797)
fails (0, -1, 2537551899)
holds (0, 0, 2537551899)
fails (0, -1, 1268775950)
holds (0, 0, 1268775950)
fails (0, -1, 634387975)
holds (0, 0, 634387975)
fails (0, -1, 317193988)
holds (0, 0, 317193988)
fails (0, -1, 158596994)
holds (0, 0, 158596994)
fails (0, -1, 79298497)
holds (0, 0, 79298497)
fails (0, -1, 39649249)
holds (0, 0, 39649249)
fails (0, -1, 19824625)
holds (0, 0, 19824625)
fails (0, -1, 9912313)
holds (0, 0, 9912313)
fails (0, -1, 4956157)
holds (0, 0, 4956157)
fails (0, -1, 2478079)
holds (0, 0, 2478079)
fails (0, -1, 1239040)
holds (0, 0, 1239040)
fails (0, -1, 619520)
holds (0, 0, 619520)
fails (0, -1, 309760)
holds (0, 0, 309760)
fails (0, -1, 154880)
holds (0, 0, 154880)
fails (0, -1, 77440)
holds (0, 0, 77440)
fails (0, -1, 38720)
holds (0, 0, 38720)
fails (0, -1, 19360)
holds (0, 0, 19360)
fails (0, -1, 9680)
holds (0, 0, 9680)
fails (0, -1, 4840)
holds (0, 0, 4840)
fails (0, -1, 2420)
holds (0, 0, 2420)
fails (0, -1, 1210)
holds (0, 0, 1210)
fails (0, -1, 605)
holds (0, 0, 605)
fails (0, -1, 303)
holds (0, 0, 303)
fails (0, -1, 152)
holds (0, 0, 152)
fails (0, -1, 76)
holds (0, 0, 76)
fails (0, -1, 38)
holds (0, 0, 38)
fails (0, -1, 19)
holds (0, 0, 19)
fails (0, -1, 10)
holds (0, 0, 10)
fails (0, -1, 5)
holds (0, 0, 5)
fails (0, -1, 3)
holds (0, 0, 3)
fails (0, -1, 2)
holds (0, 0, 2)
fails (0, -1, 1)
holds (0, 0, 1)
fails (0, -1, 0)
holds (0, 0, 0)
